vispy/geometry/triangulation.py

```diff
@@ -719,28 +640,18 @@ def _merge_duplicate_points(self):
             # rewrite edges to use i instead of j
             self.edges[self.edges == j] = i
 
-            #assert not np.any(self.edges[:,0] == self.edges[:,1])
-
             # decrement all point indexes > j
             self.edges[self.edges > j] -= 1
             dups_arr[dups_arr > j] -= 1
-            #assert not np.any(self.edges[:,0] == self.edges[:,1])
 
         self.pts = self.pts[pt_mask]
 
         # remove zero-length edges
         mask = self.edges[:, 0] != self.edges[:, 1]
         self.edges = self.edges[mask]
 
-    def _distance(self, A, B):
```

Comment:
It seems that some tests still use this method. Could you maybe remove the `_` from the name and patch any tests that use it?

Reply:
if it's currently private let's not make it public without good reason. It's likely it was private because it's not bulletproof / ready for public use